src/config.ts

```diff
@@ -108,9 +108,9 @@ export const configSchema = object({
    */
   packageName: optional(string()),
   /**
-   * Denotes types that should be generated as classes. Resolver classes can inherit from these to enforce a type contract.
+   * Denotes types that should be generated as interfaces rather than classes. Resolver classes should inherit from these to enforce a type contract.
```

Comment:
Nit: "Resolver classes should implement the interface functions to enforce a type contract".